src/index.md

```diff
@@ -8,6 +8,93 @@ styles:
 
 <div class="content-sizing" markdown="1">
 
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+build web apps that work offline.
+
+Modern offline web apps are possible thanks to **service workers** (SWs). SWs
+are background workers, written in JavaScript, that can locally save all of
+the resources needed to run your site offline. The SW also serves as the
+network proxy for your web app. In other words, in your SW code, you specify
+whether a resouce should be served from the device's local storage (AKA the
+**cache**), or from the network. Or perhaps the app should try to get a
+fresh copy of the resource from the network first, and then fall back
+to the version in the cache if the network is taking too long, and so on.
+
+The logic for determining whether to fetch resources from the cache or
+from the network is called the service worker's **caching strategy**. There
+are a lot of strategies, and which one you use depends on the resource you're
```

Comment:
"...and which one you use depends on the types or resources, how they are used and the importance of the resource."